GVRf/Framework/framework/src/main/jni/objects/textures/render_texture.cpp

```diff
@@ -23,215 +23,438 @@
 #include "eglextension/msaa/msaa.h"
 
 namespace gvr {
-RenderTexture::RenderTexture(int width, int height, GLTexture* tex) :
+GLFrameBuffer *RenderTexture::renderTexture_gl_read_buffer = nullptr;
```

Comment:
texId is being passed to readRenderResultNative. And texId is obtained from the multiview-specific RenderTexture eyeCaptureRenderTexture. So if you pass eyeCaptureRenderTexture to readRenderResultNative instead you will have the instance you need. Right?